ci-operator/step-registry/workers-rhel/ibmcloud-provision/workers-rhel-ibmcloud-provision-commands.sh

```diff
@@ -62,16 +65,17 @@ for count in $(seq 1 ${RHEL_WORKER_COUNT}); do
   idx=$(((count-1) % $computeSubnetLength))
   
   subnet=$(yq-go r "${VPC_CONFIG}" "platform.ibmcloud.computeSubnets[${idx}]")
-  zone=$(ibmcloud is subnet ${subnet} --output JSON | jq -r '.zone.name')
+  zone=$(${IBMCLOUD_CLI} is subnet ${subnet} --output JSON | jq -r '.zone.name')
   volume="${infra_id}-vol-$count"
   vmName=${infra_id}-rhel-${count}
+
   echo "computeSubnet: ${subnet}; zone: ${zone}; volume: $volume; vmName: $vmName"
 
   volumeJson=$(jq -n \
     --arg vn "$volume" \
     --argjson size ${RHEL_VM_DISK_SIZE} \
     '{"name": $vn, "volume": {"capacity": $size, "profile": {"name": "general-purpose"}}}')
-  cmd="${IBMCLOUD_CLI} is instance-create $vmName ${vpcName} ${zone} ${RHEL_VM_SIZE} ${subnet} --image ${RHEL_IMAGE} --keys ${rhelSSHKey} --sgs ${rhel_worker_sgs} --boot-volume '${volumeJson}'"
+  cmd="${IBMCLOUD_CLI} is instance-create $vmName ${vpcName} ${zone} ${RHEL_VM_SIZE} ${subnet} --image ${RHEL_IMAGE} --keys ${rhelSSHKey} --pnac-vni-sgs ${rhel_worker_sgs} --boot-volume '${volumeJson}'"
 
   echo "Creating RHEL VM..."
   run_command "${cmd}"
```

Comment:
To help debug next time, suggest to add one more line:
```
ibmcloud is instance <instance-name>
```

Reply:
in the following, 
`  ${IBMCLOUD_CLI} is instance ${vmName} --output JSON > ${insFile}     

  cat ${insFile}`